core/src/main/scala/org/apache/spark/deploy/DependencyUtils.scala

```diff
@@ -19,11 +19,11 @@ package org.apache.spark.deploy
 
 import java.io.File
 import java.net.URI
+import java.util.regex.Pattern
 
 import org.apache.commons.lang3.StringUtils
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FileSystem, Path}
-
```

Comment:
Previous import style was correct. It complied project's style

Reply:
Oops, I have pushed a commit to correct this.